src/Tgstation.Server.Host/Components/Interop/Bridge/RuntimeInformation.cs

```diff
@@ -22,7 +22,7 @@ public sealed class RuntimeInformation : ChatUpdate
 		/// <summary>
 		/// The port the HTTP server is running on.
 		/// </summary>
-		public ushort ServerPort { get; }
+		public int ServerPort { get; }
```

Comment:
Aren't ports supposed to be in `[0, 65535]` range?

Reply:
yeah but c# represents it as an `int` bro
https://docs.microsoft.com/en-us/dotnet/api/system.net.ipendpoint.port?view=net-5.0#System_Net_IPEndPoint_Port

Reply:
Whywhywhy